templates/elmPackage.dot

```diff
@@ -12,6 +12,7 @@
     ],
     "dependencies": {
         "NoRedInk/elm-decode-pipeline": "3.0.0 <= v < 4.0.0",
+        "elm-community/json-extra": "2.1.0 <= v < 3.0.0",
```

Comment:
you can add new dependencies using `elm package install ...`, but afterwards, you need to copy them to this template, as the `elm-package.json` file is generated, and the change would get wiped out